src/Generator/Generators/CSharp/CSharpCommentPrinter.cs

```diff
@@ -29,11 +29,36 @@ private static void GetCommentSections(this Comment comment, List<Section> secti
                     break;
                 case DocumentationCommentKind.BlockCommandComment:
                     var blockCommandComment = (BlockCommandComment) comment;
-                    if (blockCommandComment.CommandKind == CommentCommandKind.Return &&
-                        blockCommandComment.ParagraphComment != null)
+                    var lastBlockSection = sections.Last();
+                    if (blockCommandComment.ParagraphComment != null)
                     {
-                        sections.Add(new Section(CommentElement.Returns));
-                        blockCommandComment.ParagraphComment.GetCommentSections(sections);
+                        switch (blockCommandComment.CommandKind)
+                        {
+                            case CommentCommandKind.Brief:
+                                if (sections.Count > 1)
+                                    sections.Add(new Section(CommentElement.Summary));
+                                
+                                blockCommandComment.ParagraphComment.GetCommentSections(sections);
+                                break;
+                            case CommentCommandKind.Return:
+                                sections.Add(new Section(CommentElement.Returns));
+                                blockCommandComment.ParagraphComment.GetCommentSections(sections);
+                                break;
+                            case CommentCommandKind.Note:
```

Comment:
Notes are not the same as remarks. Remarks is a poor name ms gave to what is actually a description, there's a single such element per comment. So unless there's some other element we can map notes too, just take their text.